plink-frontend/src/app/screens/currency-list/currency-list.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="currency-header">
+  <div>Nombre</div>
+  <div>Precio</div>
+  <div>Criptomoneda</div>
+  <div>Convertir</div>
```

Comment:
use span tag

Reply:
this comment is solved in the previous PR